edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -0,0 +1,258 @@
+"""
+Tests for the exams API views
+"""
+import json
+import uuid
+from datetime import datetime
+
+import ddt
+import pytz
+from django.urls import reverse
+
+from edx_exams.apps.api.test_utils import ExamsAPITestCase
+from edx_exams.apps.api.test_utils.factories import UserFactory
+from edx_exams.apps.core.models import CourseExamConfiguration, Exam
+
+
+@ddt.ddt
+class ExamViewTests(ExamsAPITestCase):
```

Comment:
because I was pedantic about the view name this now also needs to be renamed

there is no reward for virtue